queries.sql

```diff
@@ -39,4 +39,18 @@ select count(animal_escape_attempts) from animal group by neutered;
 select min(animal_weight_kg) from animal;
 select max(animal_weight_kg) from animal;
 SELECT species, ROUND(AVG(animal_escape_attempts)) FROM animal
-  WHERE animal_date_of_birth >= '1990-01-01' AND animal_date_of_birth <= '2000-12-31' GROUP BY species;
\ No newline at end of file
+  WHERE animal_date_of_birth >= '1990-01-01' AND animal_date_of_birth <= '2000-12-31' GROUP BY species;
+
+
+select animal.owner_id, animal.animalname, owners.fullname, owners.id from animal join owners on animal.owner_id = owners.id where owners.fullname = 'Melody Pond';
+
+select animal.species_id, animal.animalname, species.id, species.fullname from animal join species on animal.species_id = species.id where species.fullname = 'Pokemon';
+
+select animal.owner_id, animal.animalname, owners.fullname, owners.id from animal join owners on animal.owner_id = owners.id where animal.owner_id = null;
+
+select count(animal.species_id), species.id, species.fullname from animal join species on animal.species_id = species.id group by species.id;
+
+select species.id, owners.id, owners.fullname from animal join species on animal.species_id  =species.id join owners on animal.owner_id = owners.id where owners.fullname = 'Jennifer Orwell';
```

Comment:
- [ ] Good job working on the queries but this does not quite fully represent the statement `List all Digimon owned by Jennifer Orwell.`

Reply:
@AnselemOdims  Can you explain what should I use in here 

Thank you